.github/workflows/node-zxc-build-release-artifact.yaml

```diff
@@ -197,24 +198,24 @@ jobs:
         uses: gradle/gradle-build-action@3fbe033aaae657f011f88f29be9e65ed26bd29ef # pin@v2
         if: ${{ inputs.version-policy == 'specified' && !cancelled() && !failure() }}
         with:
-          arguments: versionAsSpecified -PnewVersion=${{ inputs.new-version }} --scan
+          arguments: -p hedera-node versionAsSpecified -PnewVersion=${{ inputs.new-version }} --scan
```

Comment:
GitHub actions behave as before. As the 'hedera-node' build now has the folder 'hedera-node' as its root, we tell Gradle to run builds from there using the `-p` command line parameter.